Firebase device mode updates for iOS SDK v2 (#609)
rudderlabs/rudderstack-docs: docs/destinations/analytics/firebase.mdx

```
</span>
</li>
</ol>
    </TabPanel>
    <TabPanel>
        <div class="warningBlock">
          This device mode integration is supported for Firebase v8.15.0 and above.
        </div>
        Follow these steps to add Firebase to your iOS project:
<ol>
<li>Add Firebase as a destination in the <a href="https://app.rudderstack.com">RudderStack dashboard</a>.</li>
<li>Register your app in the <a href="https://console.firebase.google.com">Firebase console</a>. It will then prompt you to download the <code class="inline-code">GoogleServices-Info.plist</code> file.</li>
<li>Download and place this file in your project.</li>

> <li><code class="inline-code">RudderFirebase</code> is available through <a href="https://cocoapods.org/">CocoaPods</a>. To install it, add the following line to your <code class="inline-code">Podfile</code>:

Reviewer: ```suggestion
<li>Install <code class="inline-code">RudderFirebase</code> (available through <a href="https://cocoapods.org/">CocoaPods</a>) by adding the following line to your <code class="inline-code">Podfile</code>:
```

just to maintain the consistency to start with a  verb
